checkov/common/runners/runner_registry.py

```diff
@@ -29,11 +31,7 @@ def run(self, root_folder=None, external_checks_dir=None, files=None, guidelines
         return self.scan_reports
 
     def print_reports(self, scan_reports, args):
-        if args.output not in (
-            "github_failed_only",
-            "json",
-            "junitxml",
-        ):
+        if args.output not in OUTPUT_CHOICES:
```

Comment:
@schosterbarak why is the banner no longer displayed for `'cli'`?

And when looking at the arg parser, there is no way, this could ever by `True`.